integration/tests/cook/test_basic.py

```diff
@@ -25,42 +25,59 @@ def setUp(self):
         util.wait_for_cook(self.cook_url)
 
     def test_basic_submit(self):
-        job_uuid, resp = util.submit_job(self.cook_url)
-        self.assertEqual(resp.status_code, 201)
+        is_using_cook_executor = util.is_using_cook_executor(self.cook_url)
```

Comment:
This is repeated a few times: why not have the util method return 'cook' or 'mesos' instead of the boolean?